### PR TITLE
Add instructions to sync fork with master; change title

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

* Add instructions to regularly sync fork with master
* Changed title to include "documentation" string

### CODE REVIEW

**Review by johannes-b — APPROVED** [review]
lgtm

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Contribute to the Keptn Website
+# Contribute to the Keptn documentation
```
Comment (oleg-nenashev) [inline CONTRIBUTING.md:1]: Revert?

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Wrong title. This is a guide for the entire website, not just for the docs section

File: CONTRIBUTING.md (hunk 1 above)
Comment (StackScribe) [inline CONTRIBUTING.md:1]:
```suggestion
# Contribute to the Keptn website
```

**Review by oleg-nenashev — APPROVED** [review]

**Pull request merged**